### DISCUSSION TITLE
Can Wrapfigure be implemented

### DISCUSSION THREAD

**Discussion by mpmdean** [typst/typst#2832] [Q&A] [answered]
I would be interested in implementing figures that are tightly wrapped by text similar to the Wrapfigure in latex
https://www.overleaf.com/learn/latex/Wrapping_text_around_figures

Is this something that can currently be done?

**Comment by Enivex** [comment, marked as answer]
No, but it's on the roadmap

**Reply by mpmdean** [reply]
Sounds good. It wasn't particularly obvious to me reading the roadmap.

**Reply by wang-jinbo** [reply]
Is it usable now or not?

**Reply by Enivex** [reply]
No, but there's a workaround package for now

https://typst.app/universe/package/wrap-it
